src/core/store/external/getters.js

```diff
@@ -92,42 +92,45 @@ const getCoinGeckoTokenById = state => cgid => {
  * Get Token info including market data if exists
  */
 const contractToToken =
-  (state, getters, rootState, rootGetters) => contractAdress => {
-    contractAdress = contractAdress.toLowerCase();
-    let tokenId = platformList[contractAdress];
-    if (contractAdress === MAIN_TOKEN_ADDRESS) {
-      tokenId = rootGetters['global/network'].type.coingeckoID;
-      const networkType = rootGetters['global/network'].type;
+  (state, getters, rootState, rootGetters) => contractAddress => {
+    if (contractAddress) {
```

Comment:
can you actually figure out where it calls this without a contract address? lets find the root cause